paperqa/docs.py

```diff
@@ -778,6 +778,11 @@ async def aquery(  # noqa: PLR0912
             answer_reasoning = None
         else:
             with set_llm_session_ids(session.id):
+                iteration_context = ""
```

Comment:
[naming nit] this may be confused with `Context` (the type here)

Can we name something different?